src/components/shared/AudienceWidget.tsx

```diff
@@ -21,7 +22,12 @@ const agePerAudience = (audience: AudienceType) => {
 }
 
 export const AudienceWidget: React.FC<Props> = ({ audience }) => (
-  <div className='text-[13px] font-bold p-1 rounded-full border-2 text-orange-400 border-orange-400 size-6 flex items-center'>
+  <Text
```

Comment:
We should wrap Text with `div` to achieve exact look. We should not style and add borders on text component